h2o-py/h2o/model/model_base.py

```diff
@@ -1205,7 +1205,7 @@ def __predFor3D(self, x, y, z, colPairs, nbins, user_cols, user_num_splits):
             print("Numpy not found.  Cannot plot 2D partial plots.")
         ycol = colPairs[1]
         nBins = nbins
-        if ycol in user_cols:
+        if not(user_cols==None) and (ycol in user_cols):
```

Comment:
I think `if user_cols is not None` looks more Python-y

Reply:
and also you dont need the brackets (), python has pretty good operator precedence

Reply:
Yeap, you caught me.  Not a python coder.  Fixed.  Thanks.